flutter_vlc_player_platform_interface/lib/src/utils/options/vlc_http_options.dart

```diff
@@ -0,0 +1,36 @@
+class VlcHttpOptions {
```

Comment:
These changes look good, but are out of scope for the current PR. 
Please open a new PR and move these changes there. Some rationale would be useful, too.